src/AlgebraicGeometry/Surfaces/EnriquesAut.jl

```diff
@@ -0,0 +1,944 @@
+  io = pretty(io)
+  println(io, LowercaseOff(), "EnriquesChamber")
+  print(io, Indent())
+  show(io, MIME("text/plain"), change_base_ring(GF(2), D.tau))
+  print(io,Dedent())
+end
+
+function Base.show(io::IO, D::EnriquesChamber)
+  io = pretty(io)
+  print(io, LowercaseOff(), "EnriquesChamber")
+end
+
+@doc raw"""
+    adjacent_chamber(D::EnriquesChamber, v::ZZMatrix) -> EnriquesChamber
+
+Return return the chamber adjacent to `D` via the wall defined by `v`.
+"""
+function adjacent_chamber(D::EnriquesChamber, v::ZZMatrix)
+  gramS = D.data.gramSY
+  ref = _reflection(gramS, v)
+  tau2 = D.tau*ref
+  Dnew = EnriquesChamber(D.data, tau2, v)
+  return Dnew
+end
+
+@doc raw"""
+    rays(D::EnriquesChamber)
+    
+Return the list of primitive ray generators of ``D``.
+"""
+function rays(D::EnriquesChamber)
+  r = initial_rays(D.data)
+  return [i*D.tau for i in r]
+end
+
+@doc raw"""
+    isotropic_rays(D::EnriquesChamber)
+    
+Return the list of primitive isotropic ray generators of ``D``.
+"""
+function isotropic_rays(D::EnriquesChamber)
+  r = initial_isotropic_rays(D.data)
+  return [i*D.tau for i in r]
+end
+
+@doc raw"""
+    walls(D::EnriquesChamber)
+    
+Return the list of walls of ``D``.
+"""
+function walls(D::EnriquesChamber)
+  gramS = D.data.gramSY
+  walls0 = initial_walls(D.data)
+  return [r*D.tau for r in walls0]
+end
+
+@doc raw"""
+    hom(D1::EnriquesChamber, D2::EnriquesChamber)
+    
+Return the set of elements of ``G`` mapping ``D_1`` to ``D_2``.
+"""
+function hom(D1::EnriquesChamber, D2::EnriquesChamber)
+  result = ZZMatrix[]
+  tau1inv = inv(D1.tau)
+  t1 = change_base_ring(GF(2), tau1inv)
+  t2 = change_base_ring(GF(2), D2.tau)
+  for (i,g2) in enumerate(initial_automorphisms_mod2(D1.data))
+    h2 = t1*g2*t2
+    if D1.data.membership_test(D1.data, h2)
+      g = initial_automorphisms(D1.data)[i]
+      h = tau1inv*g*D2.tau
+      push!(result, h)
+    end
+  end
+  return result
+end
+
+@doc raw"""
+    is_congruent_with_data(D1::EnriquesChamber, D2::EnriquesChamber) -> Bool, ZZMatrix
+    
+Return whether `D1` and `D2` are congruent under ``G`` and if yes
+an element ``g \in G`` with ``D1^g=D2``.
+"""
+function is_congruent_with_data(D1::EnriquesChamber, D2::EnriquesChamber)
+  tau1inv = inv(D1.tau)
+  t1 = change_base_ring(GF(2),tau1inv)
+  t2 = change_base_ring(GF(2), D2.tau)
+  for (i,g2) in enumerate(initial_automorphisms_mod2(D1.data))
+    h2 = t1*g2*t2
+    if D1.data.membership_test(D1.data, h2)
+      g = initial_automorphisms(D1.data)[i]
+      h = tau1inv*g*D2.tau
+      return true, h
+    end
+  end
+  return false, tau1inv
+end
+
+aut(D::EnriquesChamber) = hom(D, D)
+
+@doc raw"""
+    borcherds_method(Y::EnriquesBorcherdsCtx; max_nchambers=-1)
+    
+Compute ``\mathrm{Aut}_{s}(Y)`` of the Enriques surface ``Y`` using Borcherds method. 
+
+Here ``\mathrm{Aut}_{s}(Y)`` denote the subgroup consisting of semi-symplectic automorphisms of ``Y``. 
```

Comment:
```suggestion
Here ``\mathrm{Aut}_{s}(Y)`` denotes the group consisting of semi-symplectic automorphisms of ``Y``. 
```
You do not mention subgroup of what; if of the full automorphism group, then simply "group of semi-symplectic automorphisms" should be fine ?